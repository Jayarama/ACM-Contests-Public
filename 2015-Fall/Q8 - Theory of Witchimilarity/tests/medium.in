50
Sachiko 37 81 94 blue_ncs
Spath 5 51 37 medium_tuscan_red
Garret 11 82 72 lavender_blue
Eleanore 63 82 8 zaffre
Poulsen 26 27 79 olive_drab_7
Isabelle 19 4 86 french_beige
Hasegawa 41 71 57 selective_yellow
Torgrimson 73 39 17 international_orange_aerospace
Marna 96 9 70 moccasin
Genie 29 50 17 dark_midnight_blue
Starkey 1 26 98 malachite
Reddout 34 4 53 ruddy
Tai 84 37 27 dark_sienna
Nola 62 79 88 msu_green
Coyne 71 9 40 plum_traditional
Cheri 22 18 29 tiffany_blue
Missy 92 98 5 chocolate_web
Roscoe 18 13 35 light_red_ochre
Paul 17 92 22 mardi_gras
Teed 78 47 66 persian_plum
Rattler 13 78 88 duke_blue
Ouzts 93 11 92 coquelicot
Johanna 66 1 1 laser_lemon
Wagaman 33 20 56 electric_blue
Nancee 90 46 71 deep_champagne
Sang 93 69 75 viridian
Hedy 76 99 44 taupe_gray
Robby 88 64 40 phlox
Elvin 50 2 80 cinnamon
Chaffins 83 79 40 crimson_glory
Mccreary 27 41 11 lime_color_wheel
Spear 73 77 37 smoky_black
Kera 88 38 35 utah_crimson
Hague 69 14 20 deep_ruby
Dee 90 11 34 magenta_dye
Kiana 74 81 90 honolulu_blue
Shryock 95 2 97 hot_magenta
Brouillard 37 74 66 tangerine_yellow
Rosalba 39 28 37 straw
Carmina 81 29 78 dartmouth_green
Adelia 81 7 70 fulvous
Rominger 59 93 82 green_pigment
Hee 28 99 17 khaki_html_css_khaki
Hotchkiss 51 29 38 piggy_pink
Cierra 15 6 50 vivid_violet
Harriette 3 14 16 islamic_green
Scheff 52 73 58 crimson
Delma 24 92 68 camel
Craw 74 58 46 saffron
Bibby 9 88 15 carmine_m_p
