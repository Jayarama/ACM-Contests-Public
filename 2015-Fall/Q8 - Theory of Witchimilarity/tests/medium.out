Elvin
Isabelle
Poulsen
Starkey
Wagaman
